build-cluster-ec2.yml

```diff
@@ -124,9 +248,10 @@
 
   tasks:
     - include: "bundle/tasks/install-conductr-haproxy.yml"
+    - include: "bundle/tasks/install-visualizer.yml"
 
-- name: Register ConductR with ELB
-  hosts: nodes
+- name: Register Proxy Nodes with ELB
```

Comment:
How about setting the `name` as `Register Public Nodes with ELB` for the sake of consistency with https://github.com/typesafehub/conductr-ansible/pull/34/files#diff-a1aa4b9835ceeb0000cc881144e3762aR216?